### ISSUE TITLE
distro compatibility

### ISSUE THREAD

**Issue by fossfreedom** [amrithmmh/arcMPD_budgie#1]
The readme says this is specific to Solus Budgie.  Really?  You are going to tie the theme only for Solus - or will other distro budgie desktop users able to use the theme in future?

**Comment by amrithmmh** [comment]
Yes for now its budgie.(not just solus any linux with budgie) .I am not that good with writing theme.but i will try for other DE s too

**Comment by fossfreedom** [comment]
ah - ok. Thanks for the info.  I was not inquiring about other desktop environments - just whether you were making the theme only for Solus Budgie.  

If the theme works on Manjaro Budgie, Ubuntu Budgie then great!

Perhaps maybe slightly change the wording in the README to prevent confusion?

**Comment by amrithmmh** [comment]
Ok ...do you know anyone who is good with theming especially budgie ?

**Comment by fossfreedom** [comment]
Not many folks I'm afraid :(

I've just completed the budgie desktop theme for Arc (slightly different styling from using the system theme) - https://github.com/NicoHood/arc-theme/blob/master/common/gtk-3.0/3.20/sass/_budgie.scss

Hope that helps.

**Comment by amrithmmh** [comment]
Wow...is the theme codes all handwritten?...is there any software to help ?

**Comment by fossfreedom** [comment]
yep - handwritten - no software that I'm aware of.

You can use GTK Inspector to look at applications and drill down to the styles for individual objects used.  It has a CSS editor to also allow you to craft CSS to see what impact changes will have.

**Comment by amrithmmh** [comment]
And i see most of the themes using scss/sass  ...is it possible to use part css and part scss files(ofcourse should be compiled to css) ?

**Comment by fossfreedom** [comment]
It's perhaps best to stick with one or the other - not to try and mix.

scss is simpler to code and to understand and it takes all of the chore of writing reams and reams of css.

**Comment by fossfreedom** [comment]
Fyi I have created an arc theme based for budgie desktop which you could use as a reference point https://github.com/UbuntuBudgie/pocillo-gtk-theme
